<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Real State Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="central_widget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>621</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>460</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="welcome_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Welcome to the Real Estate Manager!</string>
       </property>
       <property name="indent">
        <number>80</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddHouse"/>
    <addaction name="actionModifyHouse"/>
    <addaction name="actionSearchHouse"/>
    <addaction name="actionRemoveHouse"/>
    <addaction name="actionReport"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpContents"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAddHouse">
   <property name="text">
    <string>Add a House</string>
   </property>
  </action>
  <action name="actionModifyHouse">
   <property name="text">
    <string>Modify a House</string>
   </property>
  </action>
  <action name="actionSearchHouse">
   <property name="text">
    <string>Search a House</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>Report</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelpContents">
   <property name="text">
    <string>Help Contents</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRemoveHouse">
   <property name="text">
    <string>Remove a House</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
